integration/tests/cook/util.py

```diff
@@ -105,20 +105,62 @@ def minimal_job(**kwargs):
     return job
 
 
+def minimal_jobs(job_count, **kwargs):
+    """Build a list of of multiple homogeneous job specifications"""
+    return [ minimal_job(**kwargs) for _ in range(job_count) ]
+
+
+def submit_jobs(cook_url, job_specs, clones=1, **kwargs):
+    """
+    Create and submit multiple jobs, either cloned from a single job spec,
+    or specified individually in multiple job specs.
+    Arguments can be manually passed to the scheduler post via kwargs.
+    """
+    if isinstance(job_specs, dict):
+        job_specs = [job_specs] * clones
+    jobs = [ minimal_job(**spec) for spec in job_specs ]
+    request_body = {'jobs': jobs}
+    request_body.update(kwargs)
+    logger.info(request_body)
+    resp = session.post(f'{cook_url}/rawscheduler', json=request_body)
+    return [ j['uuid'] for j in jobs ], resp
+
+
+def kill_jobs(cook_url, jobs, assert_response=True):
+    """Kill one or more jobs"""
+    params = '&'.join(f'job={unpack_uuid(j)}' for j in jobs)
+    response = session.delete(f'{cook_url}/rawscheduler?{params}')
```

Comment:
In `query_jobs`, we pass the params as a keyword arg to `session.get`. Could we do the same for these params?